Directory.Build.targets

```diff
@@ -56,8 +48,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(IsSourceProject)' == 'true'">
-    <None Include="$(DocumentationFile)" CopyToOutputDirectory="PreserveNewest" />
-
+    <!-- Ensures docs xmls get copied to the artifacts bin output folder. -->
+    <None Include="$(XmlPath)" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
You probably want to condition this so we can still support a project that wants to ship it's source built XML.  I can imagine either a property being the pivot, or the presence of the XML file.

Reply:
Do you think I should add such condition now? Maybe we should add it when that situation arises.

Reply:
I thought you mentioned you already had that case for System.Json?  If you feel like you don't have time to solve it now you can defer it, but it seems to me like basic functionality - and was what I called out last week https://github.com/dotnet/maintenance-packages/pull/154#issuecomment-2452492908.

Reply:
It's not that I don't have time to solve it at the moment, and I agree with you that it's basic functionality. But I worry about some things:

- We aren't going to be using it right now, and I've received feedback in the past about not adding new unused/dead code.
- The reason why we're opting for using the checked-in xml files is to reduce risk and continue showing the same intellisense as in the last released packages.
- Generating intellisense from source code for the first time for these packages would not meet the bar for churn in these packages.

With the above in mind, would you still prefer that we add this functionality anyway?

Reply:
I was imagining the exercise of writing down the two behaviors (checked in vs emitted) in repo docs and then ensuring the code followed those docs might help you refine the solution.  I agree we don't want to switch anything up which was why we opted to continue checking things in for existing packages.

Reply:
From our chat conversation, with the possibility of other packages coming our way that do have a bar for taking docs changes, I think this would be an acceptable functionality to include. I'll submit another PR with it.